sdk/search/search-documents/src/logger.ts

```diff
@@ -4,6 +4,6 @@
 import { createClientLogger } from "@azure/logger";
 
 /**
- * The @azure/logger configuration for this package.
+ * The \@azure/logger configuration for this package.
```

Comment:
I feel like the backslash is a little ugly for readability. Would this work instead?

```suggestion
 * The `@azure/logger` configuration for this package.
```

Reply:
@xirzec your suggestion should work.

Reply:
Done.